js/main.js

```diff
@@ -0,0 +1,16 @@
+const GET_RANDOM_NUMBER = (min, max) => {
+  const result = (min >= 0 && min < max) ? Math.floor(Math.random() * (max - min + 1) + min) : 'Введите корректный диапозон от 0 и выше';
+  return result;
```

Comment:
Тут не нужна лишняя переменная. можешь сходу возвращать то, что вычисляешь
```suggestion
  return (min >= 0 && min < max) ? Math.floor(Math.random() * (max - min + 1) + min) : 'Введите корректный диапозон от 0 и выше';
```

Reply:
Исправил